sycl/include/CL/sycl/detail/cg.hpp

```diff
@@ -325,7 +325,8 @@ class CG {
     COPY_PTR_TO_ACC,
     COPY_ACC_TO_ACC,
     FILL,
-    UPDATE_HOST
+    UPDATE_HOST,
+    RUN_ON_HOST_INTEL
```

Comment:
Perhaps this should be named `RUN_ON_HOST` instead?

Reply:
@mkinsner @jbrodman  Could you comment on this?
I named it this way to align with name of the handler's method.